Doc/whatsnew/3.9.rst

```diff
@@ -421,11 +421,15 @@ Removed
   (Contributed by Victor Stinner in :issue:`39350`.)
 
 * The *buffering* parameter of :class:`bz2.BZ2File` has been removed. Since
-  Python 3.0, it was ignored and using it was emitting
-  :exc:`DeprecationWarning`. Pass an open file object to control how the file
-  is opened.
+  Python 3.0, it was ignored and using it emitted a :exc:`DeprecationWarning`.
+  Pass an open file object to control how the file is opened.
   (Contributed by Victor Stinner in :issue:`39357`.)
 
+* The *encoding* parameter of :func:`json.loads` has been removed.  As of
+  Python 3.1, it was deprecated and ignored.  And using it emitted a
```

Comment:
One last point: I'd suggest starting the third sentence without the "And" or combining the sentences using a semicolon. Also, after reading over it again, I realized it would be more grammatically correct to say "using it has emitted" rather than "using it emitted". 

1) As of Python 3.1, it was deprecated and ignored.  Using it has emitted a :exc:`DeprecationWarning` since Python 3.8.

2) As of Python 3.1, it was deprecated and ignored; using it has emitted a :exc:`DeprecationWarning` since Python 3.8.

Option (2) is likely more formally correct, but I think either would be fine.

Note: As a general English grammar rule, it's perfectly acceptable to start sentences with "And" (https://www.merriam-webster.com/words-at-play/words-to-not-begin-sentences-with), but it's typically discouraged to string together consecutive "and"s, even if they're separated by a period.

Reply:
In the future if you'd like me to review any documentation PRs or news entries for help with English grammar, I'd be happy to help! Feel free to request a review or @ mention me.

My primary areas of interest recently have been in asyncio, concurrent.futures, and subinterpreters, but I'm always glad to help with improving our documentation and news entries when I have the chance.

Also, I think your English skills are actually quite decent @methane, especially for someone who's native language is entirely dissimilar to English. I can generally understand the message you're trying to convey, there's just a few grammatical components that could be improved to make it more formally correct.